tests/testsuite/vendor.rs

```diff
@@ -841,6 +843,7 @@ fn git_complex() {
                     [package]
                     name = "foo"
                     version = "0.1.0"
+                    edition = "2021"
```

Comment:
Add a default edition to prevent `[WARNING] no edition set: defaulting to the 2015 edition while the latest is 2021`.